pyproject.toml

```diff
@@ -12,21 +12,20 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7"
-black = { version = ">=23.1.0", optional = true }
```

Comment:
Not using optional here is not acceptable for libraries that bundle their protobufs as it will install black, isort etc